deployment/k8s/scrumlr/templates/ingress_tls_secrets.yaml

```
@@ -0,0 +1,30 @@
+{{- if .Values.ingress.enabled }}
+{{- if .Values.ingress.secrets }}
+{{- range .Values.ingress.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .name }}
+  namespace: {{ $.Release.Name }}
+type: kubernetes.io/tls
+data:
+  tls.cert: {{ .certificate | b64enc }}
+  tls.key: {{ .key | b64enc }}
+---
+{{- end }}
+{{- end }}
+{{- if and .Values.ingress.enabled .Values.ingress.selfSigned }}
+{{- $ca := genCA "scrumlr-ca" 365 }}
+{{- $cert := genSignedCert .Values.ingress.hostname nil (list .Values.ingress.hostname) 365 $ca }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-tls" .Values.ingress.hostname }}
+  namespace: {{ .Release.Name }}
+type: kubernetes.io/tls
+data:
+  tls.cert: |- {{ $cert.Cert | nindent 4 }}
+  tls.key: |- {{ $cert.Key | nindent 4 }}
+  ca.cert: |- {{ $ca.Cert | nindent 4 }}
```

```suggestion
  tls.crt: {{ $cert.Cert | b64enc | quote }}
  tls.key: {{ $cert.Key | b64enc | quote }}
  ca.crt: {{ $ca.Cert | b64enc | quote }}
```

Fix for `illegal base64 data at input byte 0` errors